tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -62,6 +62,29 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
         }
     }
 
+    const auto& process_name = shared::GetCurrentProcessName();
+    Logger::Info("ProcessName: ", process_name);
+
+    const auto& process_command_line = shared::GetCurrentProcessCommandLine();
```

Comment:
the `&` is useless here since you return a string by value/move

Reply:
right, removed in [7a062cc](https://github.com/DataDog/dd-trace-dotnet/pull/3196/commits/7a062ccd5759da47b5c3f641a493603ed5ceeb69)